src/app/tests/TestSceneTable.cpp

```diff
@@ -787,6 +812,28 @@ void TestStoreScenes(nlTestSuite * aSuite, void * aContext)
     NL_TEST_ASSERT(aSuite, CHIP_NO_ERROR == sceneTable->GetSceneTableEntry(kFabric1, sceneId8, scene));
     NL_TEST_ASSERT(aSuite, scene == scene8);
     NL_TEST_ASSERT(aSuite, CHIP_NO_ERROR == sceneTable->SceneApplyEFS(scene));
+
+    // Test get scenes in group
```

Comment:
Yeah, this does not seem to be testing the "list buffer too small" case at all...